### DISCUSSION TITLE
QC equivalent to f-slope Type solve

### DISCUSSION THREAD

**Discussion by AKHughes1994** [ratt-ru/QuartiCal#292] [Q&A]
Hi, 

I'm making a switch from CC to QC and, as a result, I'm trying to make a (close) copy of some previously self-calibrated data. My past calibration used the K-term and the type was set to "f-slope" looking at the QC documentation the f-slope nomenclature has been retired. A prior I would guess that "delay" is the type I should be using, but I wanted to just double-check? 

Thank you,
Andrew

**Comment by JSKenyon** [comment]
Hi Andrew!

As you correctly point out, I have moved away from the old nomenclature to try and make it a little clearer how the solvers relate to physical effects. The equivalent of `f-slope` is `delay_and_offset`. This solves for a delay (a slope in frequency) and a phase offset. The newly introduced `delay` type does not include the offset, but can be useful if you want to use QuartiCal for 1GC. Hope that helps! 

Edit: Be sure to use the latest release/main branch - there were a large number of changes in `v0.2.0`.
